matchms/filtering/remove_peaks_around_precursor_mz.py

```diff
@@ -0,0 +1,35 @@
+import numpy
+from ..Spikes import Spikes
+from ..typing import SpectrumType
+
+
+def remove_peaks_around_precursor_mz(spectrum_in: SpectrumType, mz_tolerance: float = 17) -> SpectrumType:
+
+    """Remove peaks that are within mz_tolerance (in Da) of
+       the precursor mz, exlcuding the precursor peak.
+
+    Parameters
+    ----------
+    spectrum
```

Comment:
Match docstring with function spec
```suggestion
    spectrum_in
```